### DISCUSSION TITLE
Generate tractography from a t1 image

### DISCUSSION THREAD

**Discussion by josephinoo** [dipy/dipy#2459] [Tractography / Tractometry] [answered]
Hello, I am an undergraduate student, I am new to these topics, it was very helpful, if you could help me. My question is, Can you generate the streamlines with just a T1 image? Thank you very much in advance.

**Comment by ComplexNeuroScienceGroup** [comment, marked as answer]
Hi Jose! 
The answer is no, streamlines are construct from data about the water flows in the brain tissue (d-mri). 
Oh, me acabo de dar cuenta que eres de guayaquil, hola, soy de Colombia.
Bueno el asunto es que para inferir los tractos nerviosos necesitas como mencione arriba información de los flujos de agua, pero dado que la imagen t1 es estática, de esta no es posible extraer esta información.
